// vux/v2/nuxt/.nuxt/router.js
import Vue from 'vue'
import Router from 'vue-router'

Vue.use(Router)

const _496cae71 = () => import('../pages/Datetime.vue' /* webpackChunkName: "pages/Datetime" */).then(m => m.default || m)
const _0b14dac0 = () => import('../pages/XTextarea.vue.vux.js' /* webpackChunkName: "pages/XTextarea.vue.vux" */).then(m => m.default || m)
const _927be834 = () => import('../pages/XSwitch.vue.vux.js' /* webpackChunkName: "pages/XSwitch.vue.vux" */).then(m => m.default || m)
const _6f1b7c5a = () => import('../pages/PopupHeader.vue' /* webpackChunkName: "pages/PopupHeader" */).then(m => m.default || m)
const _bea1f280 = () => import('../pages/XTable.vue.vux.js' /* webpackChunkName: "pages/XTable.vue.vux" */).then(m => m.default || m)
const _478ee0d3 = () => import('../pages/Group.vue.vux.js' /* webpackChunkName: "pages/Group.vue.vux" */).then(m => m.default || m)
const _7969d913 = () => import('../pages/Issue2365.vue' /* webpackChunkName: "pages/Issue2365" */).then(m => m.default || m)
const _61cec6ea = () => import('../pages/Issue471.vue' /* webpackChunkName: "pages/Issue471" */).then(m => m.default || m)
const _0860b653 = () => import('../pages/Querystring.vue' /* webpackChunkName: "pages/Querystring" */).then(m => m.default || m)
const _6a03d32c = () => import('../pages/XTable.vue' /* webpackChunkName: "pages/XTable" */).then(m => m.default || m)
const _ad418af4 = () => import('../pages/Card.vue' /* webpackChunkName: "pages/Card" */).then(m => m.default || m)
const _583c5058 = () => import('../pages/Issue1211.vue' /* webpackChunkName: "pages/Issue1211" */).then(m => m.default || m)
const _153b0b4c = () => import('../pages/Device.vue' /* webpackChunkName: "pages/Device" */).then(m => m.default || m)
const _16e3bdd7 = () => import('../pages/Radio.vue.vux.js' /* webpackChunkName: "pages/Radio.vue.vux" */).then(m => m.default || m)
const _ec6b7c66 = () => import('../pages/XProgress.vue.vux.js' /* webpackChunkName: "pages/XProgress.vue.vux" */).then(m => m.default || m)
const _1e0facc6 = () => import('../pages/XSwitch.vue' /* webpackChunkName: "pages/XSwitch" */).then(m => m.default || m)
const _747d8079 = () => import('../pages/Divider.vue.vux.js' /* webpackChunkName: "pages/Divider.vue.vux" */).then(m => m.default || m)
const _2beda1cb = () => import('../pages/Center.vue' /* webpackChunkName: "pages/Center" */).then(m => m.default || m)
const _e5732746 = () => import('../pages/Blur.vue' /* webpackChunkName: "pages/Blur" */).then(m => m.default || m)
const _d4fb11d4 = () => import('../pages/Pullup.vue.vux.js' /* webpackChunkName: "pages/Pullup.vue.vux" */).then(m => m.default || m)
const _21061d5c = () => import('../pages/XDialog.vue.vux.js' /* webpackChunkName: "pages/XDialog.vue.vux" */).then(m => m.default || m)
const _9cec15d0 = () => import('../pages/Issue1211.vue.vux.js' /* webpackChunkName: "pages/Issue1211.vue.vux" */).then(m => m.default || m)
const _b92737be = () => import('../pages/Toast.vue' /* webpackChunkName: "pages/Toast" */).then(m => m.default || m)
const _767e0a5d = () => import('../pages/Checker.vue.vux.js' /* webpackChunkName: "pages/Checker.vue.vux" */).then(m => m.default || m)
const _5548fa62 = () => import('../pages/ButtonTab.vue.vux.js' /* webpackChunkName: "pages/ButtonTab.vue.vux" */).then(m => m.default || m)
const _96ce420e = () => import('../pages/Issue2365.vue.vux.js' /* webpackChunkName: "pages/Issue2365.vue.vux" */).then(m => m.default || m)
const _3e2acc8e = () => import('../pages/Querystring.vue.vux.js' /* webpackChunkName: "pages/Querystring.vue.vux" */).then(m => m.default || m)
const _118db18a = () => import('../pages/XImgPopup.vue' /* webpackChunkName: "pages/XImgPopup" */).then(m => m.default || m)
const _7855ad46 = () => import('../pages/PulldownPullup.vue' /* webpackChunkName: "pages/PulldownPullup" */).then(m => m.default || m)
const _5e2dc25a = () => import('../pages/FormatTime.vue' /* webpackChunkName: "pages/FormatTime" */).then(m => m.default || m)
const _0869b84c = () => import('../pages/Checklist.vue.vux.js' /* webpackChunkName: "pages/Checklist.vue.vux" */).then(m => m.default || m)
const _e45956f8 = () => import('../pages/Qrcode.vue' /* webpackChunkName: "pages/Qrcode" */).then(m => m.default || m)
const _6aa6a66c = () => import('../pages/XCircle.vue.vux.js' /* webpackChunkName: "pages/XCircle.vue.vux" */).then(m => m.default || m)
const _5a6ef1c0 = () => import('../pages/Checklist.vue' /* webpackChunkName: "pages/Checklist" */).then(m => m.default || m)
const _22d481a4 = () => import('../pages/Flow.vue' /* webpackChunkName: "pages/Flow" */).then(m => m.default || m)
const _424ae87b = () => import('../pages/XImgScroller.vue' /* webpackChunkName: "pages/XImgScroller" */).then(m => m.default || m)
const _d9f85b30 = () => import('../pages/Qrcode.vue.vux.js' /* webpackChunkName: "pages/Qrcode.vue.vux" */).then(m => m.default || m)
const _52b41bb7 = () => import('../pages/Issue833.vue.vux.js' /* webpackChunkName: "pages/Issue833.vue.vux" */).then(m => m.default || m)
const _f09721a8 = () => import('../pages/Flexbox.vue' /* webpackChunkName: "pages/Flexbox" */).then(m => m.default || m)
const _18d2b49c = () => import('../pages/XHr.vue' /* webpackChunkName: "pages/XHr" */).then(m => m.default || m)
const _b3772922 = () => import('../pages/Pulldown.vue.vux.js' /* webpackChunkName: "pages/Pulldown.vue.vux" */).then(m => m.default || m)
const _34374538 = () => import('../pages/Issue649.vue' /* webpackChunkName: "pages/Issue649" */).then(m => m.default || m)
const _f1ab4f08 = () => import('../pages/InlineCalendarEachDaySlot.vue' /* webpackChunkName: "pages/InlineCalendarEachDaySlot" */).then(m => m.default || m)
const _bf1b0226 = () => import('../pages/XHeader.vue.vux.js' /* webpackChunkName: "pages/XHeader.vue.vux" */).then(m => m.default || m)
const _0017bf32 = () => import('../pages/Drawer.vue' /* webpackChunkName: "pages/Drawer" */).then(m => m.default || m)
const _bd8099fe = () => import('../pages/XIcon.vue.vux.js' /* webpackChunkName: "pages/XIcon.vue.vux" */).then(m => m.default || m)
const _402ae4e7 = () => import('../pages/Donate.vue.vux.js' /* webpackChunkName: "pages/Donate.vue.vux" */).then(m => m.default || m)
const _17522a80 = () => import('../pages/Flexbox.vue.vux.js' /* webpackChunkName: "pages/Flexbox.vue.vux" */).then(m => m.default || m)
const _7c2c7796 = () => import('../pages/Pullup.vue' /* webpackChunkName: "pages/Pullup" */).then(m => m.default || m)
const _04450158 = () => import('../pages/PopupPicker.vue' /* webpackChunkName: "pages/PopupPicker" */).then(m => m.default || m)
const _17d9bc5a = () => import('../pages/XAddress.vue.vux.js' /* webpackChunkName: "pages/XAddress.vue.vux" */).then(m => m.default || m)
const _1514c9d2 = () => import('../pages/ToggleText.vue' /* webpackChunkName: "pages/ToggleText" */).then(m => m.default || m)
const _16ae82e5 = () => import('../pages/Donate.vue' /* webpackChunkName: "pages/Donate" */).then(m => m.default || m)
const _b2d94b22 = () => import('../pages/InlineLoading.vue.vux.js' /* webpackChunkName: "pages/InlineLoading.vue.vux" */).then(m => m.default || m)
const _77015b72 = () => import('../pages/Close.vue' /* webpackChunkName: "pages/Close" */).then(m => m.default || m)
const _450538f4 = () => import('../pages/TabbarSimple.vue.vux.js' /* webpackChunkName: "pages/TabbarSimple.vue.vux" */).then(m => m.default || m)
const _7fde83c1 = () => import('../pages/ColorPicker.vue.vux.js' /* webpackChunkName: "pages/ColorPicker.vue.vux" */).then(m => m.default || m)
const _c10b0fb6 = () => import('../pages/PopupRadio.vue' /* webpackChunkName: "pages/PopupRadio" */).then(m => m.default || m)
const _23668cf0 = () => import('../pages/OnePx.vue' /* webpackChunkName: "pages/OnePx" */).then(m => m.default || m)
const _586145a2 = () => import('../pages/Badge.vue.vux.js' /* webpackChunkName: "pages/Badge.vue.vux" */).then(m => m.default || m)
const _f7d87546 = () => import('../pages/Pulldown.vue' /* webpackChunkName: "pages/Pulldown" */).then(m => m.default || m)
const _c418e76c = () => import('../pages/Swipeout.vue' /* webpackChunkName: "pages/Swipeout" */).then(m => m.default || m)
const _a4e474fa = () => import('../pages/CellBox.vue' /* webpackChunkName: "pages/CellBox" */).then(m => m.default || m)
const _04726126 = () => import('../pages/InlineCalendar.vue' /* webpackChunkName: "pages/InlineCalendar" */).then(m => m.default || m)
const _42a0c56d = () => import('../pages/Popover.vue.vux.js' /* webpackChunkName: "pages/Popover.vue.vux" */).then(m => m.default || m)
const _05448fba = () => import('../pages/Confirm.vue' /* webpackChunkName: "pages/Confirm" */).then(m => m.default || m)
const _6f01770f = () => import('../pages/Tab.vue' /* webpackChunkName: "pages/Tab" */).then(m => m.default || m)
const _54fa525c = () => import('../pages/XAddress.vue' /* webpackChunkName: "pages/XAddress" */).then(m => m.default || m)
const _1bb388fb = () => import('../pages/Msg.vue' /* webpackChunkName: "pages/Msg" */).then(m => m.default || m)
const _20911173 = () => import('../pages/Demo.vue.vux.js' /* webpackChunkName: "pages/Demo.vue.vux" */).then(m => m.default || m)
const _b0da9956 = () => import('../pages/Timeline.vue.vux.js' /* webpackChunkName: "pages/Timeline.vue.vux" */).then(m => m.default || m)
const _2784dbcf = () => import('../pages/IconLoading.vue.vux.js' /* webpackChunkName: "pages/IconLoading.vue.vux" */).then(m => m.default || m)
const _7bd21764 = () => import('../pages/Test.vue.vux.js' /* webpackChunkName: "pages/Test.vue.vux" */).then(m => m.default || m)
const _ee4fa5d2 = () => import('../pages/Clocker.vue.vux.js' /* webpackChunkName: "pages/Clocker.vue.vux" */).then(m => m.default || m)
const _4d398179 = () => import('../pages/Spinner.vue.vux.js' /* webpackChunkName: "pages/Spinner.vue.vux" */).then(m => m.default || m)
const _823a4394 = () => import('../pages/DatetimeView.vue' /* webpackChunkName: "pages/DatetimeView" */).then(m => m.default || m)
const _191a0dc6 = () => import('../pages/Icon.vue.vux.js' /* webpackChunkName: "pages/Icon.vue.vux" */).then(m => m.default || m)
const _4ec8eefc = () => import('../pages/InlineXNumber.vue' /* webpackChunkName: "pages/InlineXNumber" */).then(m => m.default || m)
const _d05e4720 = () => import('../pages/InlineCalendarMulti.vue.vux.js' /* webpackChunkName: "pages/InlineCalendarMulti.vue.vux" */).then(m => m.default || m)
const _6c20813d = () => import('../pages/WechatEmotion.vue.vux.js' /* webpackChunkName: "pages/WechatEmotion.vue.vux" */).then(m => m.default || m)
const _ddefc7e2 = () => import('../pages/Masker.vue' /* webpackChunkName: "pages/Masker" */).then(m => m.default || m)
const _78df1b7c = () => import('../pages/Inline-calendar-start-date.vue.vux.js' /* webpackChunkName: "pages/Inline-calendar-start-date.vue.vux" */).then(m => m.default || m)
const _42b8fe38 = () => import('../pages/OnePx.vue.vux.js' /* webpackChunkName: "pages/OnePx.vue.vux" */).then(m => m.default || m)
const _01faef7c = () => import('../pages/Step.vue' /* webpackChunkName: "pages/Step" */).then(m => m.default || m)
const _5ba3d656 = () => import('../pages/Selector.vue' /* webpackChunkName: "pages/Selector" */).then(m => m.default || m)
const _66ad8a6e = () => import('../pages/TabbarLink.vue' /* webpackChunkName: "pages/TabbarLink" */).then(m => m.default || m)
const _2ab9d46f = () => import('../pages/Previewer.vue' /* webpackChunkName: "pages/Previewer" */).then(m => m.default || m)
const _7dc49ed8 = () => import('../pages/Cell.vue.vux.js' /* webpackChunkName: "pages/Cell.vue.vux" */).then(m => m.default || m)
const _68639451 = () => import('../pages/LoadMore.vue' /* webpackChunkName: "pages/LoadMore" */).then(m => m.default || m)
const _18f88126 = () => import('../pages/TabbarIcon.vue' /* webpackChunkName: "pages/TabbarIcon" */).then(m => m.default || m)
const _dc15ccf8 = () => import('../pages/Picker.vue' /* webpackChunkName: "pages/Picker" */).then(m => m.default || m)
const _72ea037c = () => import('../pages/InlineCalendarMulti.vue' /* webpackChunkName: "pages/InlineCalendarMulti" */).then(m => m.default || m)
const _1f4a6d68 = () => import('../pages/Picker.vue.vux.js' /* webpackChunkName: "pages/Picker.vue.vux" */).then(m => m.default || m)
const _510ea8fc = () => import('../pages/DatetimeRange.vue' /* webpackChunkName: "pages/DatetimeRange" */).then(m => m.default || m)
const _c4c8cd94 = () => import('../pages/Issue702.vue.vux.js' /* webpackChunkName: "pages/Issue702.vue.vux" */).then(m => m.default || m)
const _3a2a2794 = () => import('../pages/Loading.vue' /* webpackChunkName: "pages/Loading" */).then(m => m.default || m)
const _ba4280b6 = () => import('../pages/GridNumber.vue' /* webpackChunkName: "pages/GridNumber" */).then(m => m.default || m)
const _30e46621 = () => import('../pages/XImg.vue' /* webpackChunkName: "pages/XImg" */).then(m => m.default || m)
const _2da079f8 = () => import('../pages/Issue865.vue.vux.js' /* webpackChunkName: "pages/Issue865.vue.vux" */).then(m => m.default || m)
const _328b7f6f = () => import('../pages/Issue189.vue' /* webpackChunkName: "pages/Issue189" */).then(m => m.default || m)
const _f9f1fd84 = () => import('../pages/TabbarLink.vue.vux.js' /* webpackChunkName: "pages/TabbarLink.vue.vux" */).then(m => m.default || m)
const _5db332ef = () => import('../pages/ScrollerHeader.vue.vux.js' /* webpackChunkName: "pages/ScrollerHeader.vue.vux" */).then(m => m.default || m)
const _bd95e218 = () => import('../pages/Issue865.vue' /* webpackChunkName: "pages/Issue865" */).then(m => m.default || m)
const _e7fd051e = () => import('../pages/CheckIcon.vue.vux.js' /* webpackChunkName: "pages/CheckIcon.vue.vux" */).then(m => m.default || m)
const _29859580 = () => import('../pages/Shake.vue' /* webpackChunkName: "pages/Shake" */).then(m => m.default || m)
const _854a384a = () => import('../pages/Datetime.vue.vux.js' /* webpackChunkName: "pages/Datetime.vue.vux" */).then(m => m.default || m)
const _d08f4fa0 = () => import('../pages/Grid.vue.vux.js' /* webpackChunkName: "pages/Grid.vue.vux" */).then(m => m.default || m)
const _1b71efb0 = () => import('../pages/Scroller.vue' /* webpackChunkName: "pages/Scroller" */).then(m => m.default || m)
const _3c98b4d6 = () => import('../pages/Alert.vue' /* webpackChunkName: "pages/Alert" */).then(m => m.default || m)
const _4b3887e4 = () => import('../pages/Search.vue.vux.js' /* webpackChunkName: "pages/Search.vue.vux" */).then(m => m.default || m)
const _66f68c37 = () => import('../pages/Range.vue' /* webpackChunkName: "pages/Range" */).then(m => m.default || m)
const _59e170c3 = () => import('../pages/NumberRoller.vue.vux.js' /* webpackChunkName: "pages/NumberRoller.vue.vux" */).then(m => m.default || m)
const _80d1c956 = () => import('../pages/Range.vue.vux.js' /* webpackChunkName: "pages/Range.vue.vux" */).then(m => m.default || m)
const _18b196e6 = () => import('../pages/WepayInput.vue.vux.js' /* webpackChunkName: "pages/WepayInput.vue.vux" */).then(m => m.default || m)
const _6a47bf08 = () => import('../pages/Countup.vue.vux.js' /* webpackChunkName: "pages/Countup.vue.vux" */).then(m => m.default || m)
const _8316ef06 = () => import('../pages/Masker.vue.vux.js' /* webpackChunkName: "pages/Masker.vue.vux" */).then(m => m.default || m)
const _3b2656a4 = () => import('../pages/WeekCalendar.vue.vux.js' /* webpackChunkName: "pages/WeekCalendar.vue.vux" */).then(m => m.default || m)
const _4d60d027 = () => import('../pages/GridNumber.vue.vux.js' /* webpackChunkName: "pages/GridNumber.vue.vux" */).then(m => m.default || m)
const _5ddb5db0 = () => import('../pages/XButton.vue.vux.js' /* webpackChunkName: "pages/XButton.vue.vux" */).then(m => m.default || m)
const _1e49e0a6 = () => import('../pages/ScrollerFull.vue.vux.js' /* webpackChunkName: "pages/ScrollerFull.vue.vux" */).then(m => m.default || m)
const _1159ce2b = () => import('../pages/Countdown.vue' /* webpackChunkName: "pages/Countdown" */).then(m => m.default || m)
const _07cb2d91 = () => import('../pages/Issue598.vue' /* webpackChunkName: "pages/Issue598" */).then(m => m.default || m)
const _4fad1e39 = () => import('../pages/PopupHeader.vue.vux.js' /* webpackChunkName: "pages/PopupHeader.vue.vux" */).then(m => m.default || m)
const _5baf06f0 = () => import('../pages/DatetimeRange.vue.vux.js' /* webpackChunkName: "pages/DatetimeRange.vue.vux" */).then(m => m.default || m)
const _6aa520bb = () => import('../pages/Issue598.vue.vux.js' /* webpackChunkName: "pages/Issue598.vue.vux" */).then(m => m.default || m)
const _eb0612f4 = () => import('../pages/Agree.vue.vux.js' /* webpackChunkName: "pages/Agree.vue.vux" */).then(m => m.default || m)
const _29e4ff44 = () => import('../pages/FormPreview.vue' /* webpackChunkName: "pages/FormPreview" */).then(m => m.default || m)
const _088e7b4f = () => import('../pages/WechatEmotion.vue' /* webpackChunkName: "pages/WechatEmotion" */).then(m => m.default || m)
const _c2ad4714 = () => import('../pages/DatetimeView.vue.vux.js' /* webpackChunkName: "pages/DatetimeView.vue.vux" */).then(m => m.default || m)
const _3d1af62f = () => import('../pages/Checker.vue' /* webpackChunkName: "pages/Checker" */).then(m => m.default || m)
const _0f058682 = () => import('../pages/XProgress.vue' /* webpackChunkName: "pages/XProgress" */).then(m => m.default || m)
const _09fb3886 = () => import('../pages/Popup.vue.vux.js' /* webpackChunkName: "pages/Popup.vue.vux" */).then(m => m.default || m)
const _bcad1fe8 = () => import('../pages/Rater.vue' /* webpackChunkName: "pages/Rater" */).then(m => m.default || m)
const _5a72119b = () => import('../pages/CheckIcon.vue' /* webpackChunkName: "pages/CheckIcon" */).then(m => m.default || m)
const _f13a8400 = () => import('../pages/Issue461.vue.vux.js' /* webpackChunkName: "pages/Issue461.vue.vux" */).then(m => m.default || m)
const _289403ea = () => import('../pages/XIcon.vue' /* webpackChunkName: "pages/XIcon" */).then(m => m.default || m)
const _7586f0b2 = () => import('../pages/PopupRadio.vue.vux.js' /* webpackChunkName: "pages/PopupRadio.vue.vux" */).then(m => m.default || m)
const _7c99aade = () => import('../pages/XImgScroller.vue.vux.js' /* webpackChunkName: "pages/XImgScroller.vue.vux" */).then(m => m.default || m)
const _2517e46f = () => import('../pages/Blur.vue.vux.js' /* webpackChunkName: "pages/Blur.vue.vux" */).then(m => m.default || m)
const _3038730e = () => import('../pages/Reddot.vue' /* webpackChunkName: "pages/Reddot" */).then(m => m.default || m)
const _3fc047d2 = () => import('../pages/FormatTime.vue.vux.js' /* webpackChunkName: "pages/FormatTime.vue.vux" */).then(m => m.default || m)
const _9bd4e314 = () => import('../pages/Loading.vue.vux.js' /* webpackChunkName: "pages/Loading.vue.vux" */).then(m => m.default || m)
const _a9805042 = () => import('../pages/InlineXSwitch.vue.vux.js' /* webpackChunkName: "pages/InlineXSwitch.vue.vux" */).then(m => m.default || m)
const _a0985578 = () => import('../pages/XInput.vue.vux.js' /* webpackChunkName: "pages/XInput.vue.vux" */).then(m => m.default || m)
const _0496c4aa = () => import('../pages/InlineXNumber.vue.vux.js' /* webpackChunkName: "pages/InlineXNumber.vue.vux" */).then(m => m.default || m)
const _1bc93c38 = () => import('../pages/Calendar.vue.vux.js' /* webpackChunkName: "pages/Calendar.vue.vux" */).then(m => m.default || m)
const _610ffd73 = () => import('../pages/Comment.vue.vux.js' /* webpackChunkName: "pages/Comment.vue.vux" */).then(m => m.default || m)
const _a166f56a = () => import('../pages/ColorPicker.vue' /* webpackChunkName: "pages/ColorPicker" */).then(m => m.default || m)
const _d9a8f3b4 = () => import('../pages/Card.vue.vux.js' /* webpackChunkName: "pages/Card.vue.vux" */).then(m => m.default || m)
const _21d4bc63 = () => import('../pages/Actionsheet.vue' /* webpackChunkName: "pages/Actionsheet" */).then(m => m.default || m)
const _61e77976 = () => import('../pages/Issue702.vue' /* webpackChunkName: "pages/Issue702" */).then(m => m.default || m)
const _34c98360 = () => import('../pages/CellFormPreview.vue.vux.js' /* webpackChunkName: "pages/CellFormPreview.vue.vux" */).then(m => m.default || m)
const _7d65dbd0 = () => import('../pages/Wechat.vue.vux.js' /* webpackChunkName: "pages/Wechat.vue.vux" */).then(m => m.default || m)
const _4f56b7c6 = () => import('../pages/Previewer.vue.vux.js' /* webpackChunkName: "pages/Previewer.vue.vux" */).then(m => m.default || m)
const _131552ee = () => import('../pages/NumberRoller.vue' /* webpackChunkName: "pages/NumberRoller" */).then(m => m.default || m)
const _50ea876f = () => import('../pages/PulldownPullup.vue.vux.js' /* webpackChunkName: "pages/PulldownPullup.vue.vux" */).then(m => m.default || m)
const _0fb681c6 = () => import('../pages/WepayInput.vue' /* webpackChunkName: "pages/WepayInput" */).then(m => m.default || m)
const _afb1b152 = () => import('../pages/Issue624.vue.vux.js' /* webpackChunkName: "pages/Issue624.vue.vux" */).then(m => m.default || m)
const _496f1226 = () => import('../pages/InlineXSwitch.vue' /* webpackChunkName: "pages/InlineXSwitch" */).then(m => m.default || m)
const _6ee9975e = () => import('../pages/GridComponent.vue' /* webpackChunkName: "pages/GridComponent" */).then(m => m.default || m)
const _75e5c481 = () => import('../pages/Issue471.vue.vux.js' /* webpackChunkName: "pages/Issue471.vue.vux" */).then(m => m.default || m)
const _91f62142 = () => import('../pages/InlineCalendar.vue.vux.js' /* webpackChunkName: "pages/InlineCalendar.vue.vux" */).then(m => m.default || m)
const _65827308 = () => import('../pages/Test.vue' /* webpackChunkName: "pages/Test" */).then(m => m.default || m)
const _a0260bde = () => import('../pages/Msg.vue.vux.js' /* webpackChunkName: "pages/Msg.vue.vux" */).then(m => m.default || m)
const _509f203c = () => import('../pages/Swipeout.vue.vux.js' /* webpackChunkName: "pages/Swipeout.vue.vux" */).then(m => m.default || m)
const _6f097426 = () => import('../pages/Popup.vue' /* webpackChunkName: "pages/Popup" */).then(m => m.default || m)
const _08d37d4e = () => import('../pages/FormPreview.vue.vux.js' /* webpackChunkName: "pages/FormPreview.vue.vux" */).then(m => m.default || m)
const _86cf4860 = () => import('../pages/Marquee.vue.vux.js' /* webpackChunkName: "pages/Marquee.vue.vux" */).then(m => m.default || m)
const _4627c84e = () => import('../pages/Demo.vue' /* webpackChunkName: "pages/Demo" */).then(m => m.default || m)
const _11b14b9d = () => import('../pages/Badge.vue' /* webpackChunkName: "pages/Badge" */).then(m => m.default || m)
const _2fd9ff5f = () => import('../pages/TabbarIcon.vue.vux.js' /* webpackChunkName: "pages/TabbarIcon.vue.vux" */).then(m => m.default || m)
const _6ae24275 = () => import('../pages/DemoList.vue.vux.js' /* webpackChunkName: "pages/DemoList.vue.vux" */).then(m => m.default || m)
const _ff541792 = () => import('../pages/Timeline.vue' /* webpackChunkName: "pages/Timeline" */).then(m => m.default || m)
const _44f1fe06 = () => import('../pages/IconLoading.vue' /* webpackChunkName: "pages/IconLoading" */).then(m => m.default || m)
const _57f1bdec = () => import('../pages/Shake.vue.vux.js' /* webpackChunkName: "pages/Shake.vue.vux" */).then(m => m.default || m)
const _3d4775ea = () => import('../pages/Inline-calendar-start-date.vue' /* webpackChunkName: "pages/Inline-calendar-start-date" */).then(m => m.default || m)
const _48848dbc = () => import('../pages/Scroller.vue.vux.js' /* webpackChunkName: "pages/Scroller.vue.vux" */).then(m => m.default || m)
const _2888d184 = () => import('../pages/Swiper.vue.vux.js' /* webpackChunkName: "pages/Swiper.vue.vux" */).then(m => m.default || m)
const _dc6da440 = () => import('../pages/Rater.vue.vux.js' /* webpackChunkName: "pages/Rater.vue.vux" */).then(m => m.default || m)
const _bf11c542 = () => import('../pages/Sticky.vue' /* webpackChunkName: "pages/Sticky" */).then(m => m.default || m)
const _c89f25a6 = () => import('../pages/Sticky.vue.vux.js' /* webpackChunkName: "pages/Sticky.vue.vux" */).then(m => m.default || m)
const _28d1a926 = () => import('../pages/TabbarSimple.vue' /* webpackChunkName: "pages/TabbarSimple" */).then(m => m.default || m)
const _aebcf8c2 = () => import('../pages/XHeader.vue' /* webpackChunkName: "pages/XHeader" */).then(m => m.default || m)
const _34048cea = () => import('../pages/Step.vue.vux.js' /* webpackChunkName: "pages/Step.vue.vux" */).then(m => m.default || m)
const _5ed77ee1 = () => import('../pages/Countdown.vue.vux.js' /* webpackChunkName: "pages/Countdown.vue.vux" */).then(m => m.default || m)
const _300f6940 = () => import('../pages/SearchStatic.vue.vux.js' /* webpackChunkName: "pages/SearchStatic.vue.vux" */).then(m => m.default || m)
const _643d13b4 = () => import('../pages/Agree.vue' /* webpackChunkName: "pages/Agree" */).then(m => m.default || m)
const _732bcfd8 = () => import('../pages/Cell.vue' /* webpackChunkName: "pages/Cell" */).then(m => m.default || m)
const _875b4cc4 = () => import('../pages/Search.vue' /* webpackChunkName: "pages/Search" */).then(m => m.default || m)
const _7bd2bcae = () => import('../pages/Actionsheet.vue.vux.js' /* webpackChunkName: "pages/Actionsheet.vue.vux" */).then(m => m.default || m)
const _01123120 = () => import('../pages/InlineCalendarEachDaySlot.vue.vux.js' /* webpackChunkName: "pages/InlineCalendarEachDaySlot.vue.vux" */).then(m => m.default || m)
const _3a4a4038 = () => import('../pages/Uploader.vue' /* webpackChunkName: "pages/Uploader" */).then(m => m.default || m)
const _d3f72faa = () => import('../pages/Toast.vue.vux.js' /* webpackChunkName: "pages/Toast.vue.vux" */).then(m => m.default || m)
const _5690e34c = () => import('../pages/XTextarea.vue' /* webpackChunkName: "pages/XTextarea" */).then(m => m.default || m)
const _add6b160 = () => import('../pages/XHr.vue.vux.js' /* webpackChunkName: "pages/XHr.vue.vux" */).then(m => m.default || m)
const _b4336a42 = () => import('../pages/ScrollerFull.vue' /* webpackChunkName: "pages/ScrollerFull" */).then(m => m.default || m)
const _1b62be5b = () => import('../pages/XNumber.vue' /* webpackChunkName: "pages/XNumber" */).then(m => m.default || m)
const _16821c24 = () => import('../pages/ScrollerSwiper.vue.vux.js' /* webpackChunkName: "pages/ScrollerSwiper.vue.vux" */).then(m => m.default || m)
const _03c4e5c2 = () => import('../pages/Popover.vue' /* webpackChunkName: "pages/Popover" */).then(m => m.default || m)
const _778e6aa0 = () => import('../pages/Device.vue.vux.js' /* webpackChunkName: "pages/Device.vue.vux" */).then(m => m.default || m)
const _ece9add2 = () => import('../pages/DemoList.vue' /* webpackChunkName: "pages/DemoList" */).then(m => m.default || m)
const _33e6262b = () => import('../pages/XImg.vue.vux.js' /* webpackChunkName: "pages/XImg.vue.vux" */).then(m => m.default || m)
const _497b901c = () => import('../pages/Wechat.vue' /* webpackChunkName: "pages/Wechat" */).then(m => m.default || m)
const _bfb89346 = () => import('../pages/InlineLoading.vue' /* webpackChunkName: "pages/InlineLoading" */).then(m => m.default || m)
const _4d6ca31c = () => import('../pages/Confirm.vue.vux.js' /* webpackChunkName: "pages/Confirm.vue.vux" */).then(m => m.default || m)
const _0255b46e = () => import('../pages/Swiper.vue' /* webpackChunkName: "pages/Swiper" */).then(m => m.default || m)
const _3f966e4c = () => import('../pages/XDialog.vue' /* webpackChunkName: "pages/XDialog" */).then(m => m.default || m)
const _3b66a17e = () => import('../pages/Panel.vue' /* webpackChunkName: "pages/Panel" */).then(m => m.default || m)
const _5e77fd9c = () => import('../pages/CellFormPreview.vue' /* webpackChunkName: "pages/CellFormPreview" */).then(m => m.default || m)
const _6e1eafd9 = () => import('../pages/Comment.vue' /* webpackChunkName: "pages/Comment" */).then(m => m.default || m)
const _4e123c66 = () => import('../pages/AppTest.vue' /* webpackChunkName: "pages/AppTest" */).then(m => m.default || m)
const _5548d82e = () => import('../pages/Panel.vue.vux.js' /* webpackChunkName: "pages/Panel.vue.vux" */).then(m => m.default || m)
const _2c98df16 = () => import('../pages/ToggleText.vue.vux.js' /* webpackChunkName: "pages/ToggleText.vue.vux" */).then(m => m.default || m)
const _4f6a4870 = () => import('../pages/WeekCalendar.vue' /* webpackChunkName: "pages/WeekCalendar" */).then(m => m.default || m)
const _9276585e = () => import('../pages/XImgPopup.vue.vux.js' /* webpackChunkName: "pages/XImgPopup.vue.vux" */).then(m => m.default || m)
const _ad232446 = () => import('../pages/ScrollerHeader.vue' /* webpackChunkName: "pages/ScrollerHeader" */).then(m => m.default || m)
const _3b5ed879 = () => import('../pages/Group.vue' /* webpackChunkName: "pages/Group" */).then(m => m.default || m)
const _7de11f6f = () => import('../pages/Icon.vue' /* webpackChunkName: "pages/Icon" */).then(m => m.default || m)
const _3b35be70 = () => import('../pages/NotFoundComponent.vue' /* webpackChunkName: "pages/NotFoundComponent" */).then(m => m.default || m)
const _a8b52198 = () => import('../pages/Issue649.vue.vux.js' /* webpackChunkName: "pages/Issue649.vue.vux" */).then(m => m.default || m)
const _555044f7 = () => import('../pages/Selector.vue.vux.js' /* webpackChunkName: "pages/Selector.vue.vux" */).then(m => m.default || m)
const _762ac154 = () => import('../pages/Tabbar.vue' /* webpackChunkName: "pages/Tabbar" */).then(m => m.default || m)
const _4fcfa1fb = () => import('../pages/LoadMore.vue.vux.js' /* webpackChunkName: "pages/LoadMore.vue.vux" */).then(m => m.default || m)
const _65387828 = () => import('../pages/Issue461.vue' /* webpackChunkName: "pages/Issue461" */).then(m => m.default || m)
const _1beba148 = () => import('../pages/Flow.vue.vux.js' /* webpackChunkName: "pages/Flow.vue.vux" */).then(m => m.default || m)
const _423540ac = () => import('../pages/SearchStatic.vue' /* webpackChunkName: "pages/SearchStatic" */).then(m => m.default || m)
const _a1251e02 = () => import('../pages/AppTest.vue.vux.js' /* webpackChunkName: "pages/AppTest.vue.vux" */).then(m => m.default || m)
const _46489128 = () => import('../pages/XInput.vue' /* webpackChunkName: "pages/XInput" */).then(m => m.default || m)
const _4821edc4 = () => import('../pages/XButton.vue' /* webpackChunkName: "pages/XButton" */).then(m => m.default || m)
const _50af5b98 = () => import('../pages/Tabbar.vue.vux.js' /* webpackChunkName: "pages/Tabbar.vue.vux" */).then(m => m.default || m)
const _1d626506 = () => import('../pages/Tab.vue.vux.js' /* webpackChunkName: "pages/Tab.vue.vux" */).then(m => m.default || m)
const _0feabb08 = () => import('../pages/Uploader.vue.vux.js' /* webpackChunkName: "pages/Uploader.vue.vux" */).then(m => m.default || m)
const _13e654b4 = () => import('../pages/Calendar.vue' /* webpackChunkName: "pages/Calendar" */).then(m => m.default || m)
const _2af71489 = () => import('../pages/CellBox.vue.vux.js' /* webpackChunkName: "pages/CellBox.vue.vux" */).then(m => m.default || m)
const _4e25de96 = () => import('../pages/Clocker.vue' /* webpackChunkName: "pages/Clocker" */).then(m => m.default || m)
const _36638c06 = () => import('../pages/ButtonTab.vue' /* webpackChunkName: "pages/ButtonTab" */).then(m => m.default || m)
const _d7b9f2da = () => import('../pages/Spinner.vue' /* webpackChunkName: "pages/Spinner" */).then(m => m.default || m)
const _c80b53d6 = () => import('../pages/Issue833.vue' /* webpackChunkName: "pages/Issue833" */).then(m => m.default || m)
const _865b8088 = () => import('../pages/Grid.vue' /* webpackChunkName: "pages/Grid" */).then(m => m.default || m)
const _3c0add98 = () => import('../pages/PopupPicker.vue.vux.js' /* webpackChunkName: "pages/PopupPicker.vue.vux" */).then(m => m.default || m)
const _59eed7f5 = () => import('../pages/Radio.vue' /* webpackChunkName: "pages/Radio" */).then(m => m.default || m)
const _1018f8b1 = () => import('../pages/XNumber.vue.vux.js' /* webpackChunkName: "pages/XNumber.vue.vux" */).then(m => m.default || m)
const _0746cdc6 = () => import('../pages/Issue189.vue.vux.js' /* webpackChunkName: "pages/Issue189.vue.vux" */).then(m => m.default || m)
const _30871e75 = () => import('../pages/Issue624.vue' /* webpackChunkName: "pages/Issue624" */).then(m => m.default || m)
const _b3cddfc8 = () => import('../pages/Marquee.vue' /* webpackChunkName: "pages/Marquee" */).then(m => m.default || m)
const _11065c4d = () => import('../pages/Thanks.vue.vux.js' /* webpackChunkName: "pages/Thanks.vue.vux" */).then(m => m.default || m)
const _1ee6f73c = () => import('../pages/XCircle.vue' /* webpackChunkName: "pages/XCircle" */).then(m => m.default || m)
const _170a9a40 = () => import('../pages/ViewBox.vue' /* webpackChunkName: "pages/ViewBox" */).then(m => m.default || m)
const _6392d1d6 = () => import('../pages/Alert.vue.vux.js' /* webpackChunkName: "pages/Alert.vue.vux" */).then(m => m.default || m)
const _afdc1b82 = () => import('../pages/Thanks.vue' /* webpackChunkName: "pages/Thanks" */).then(m => m.default || m)
const _c86030da = () => import('../pages/Divider.vue' /* webpackChunkName: "pages/Divider" */).then(m => m.default || m)
const _c54b4838 = () => import('../pages/Countup.vue' /* webpackChunkName: "pages/Countup" */).then(m => m.default || m)
const _4f3a4648 = () => import('../pages/ScrollerSwiper.vue' /* webpackChunkName: "pages/ScrollerSwiper" */).then(m => m.default || m)
const _fc57fc0a = () => import('../pages/GridComponent.vue.vux.js' /* webpackChunkName: "pages/GridComponent.vue.vux" */).then(m => m.default || m)



if (process.client) {
  window.history.scrollRestoration = 'manual'
}
const scrollBehavior = function (to, from, savedPosition) {
  // if the returned position is falsy or an empty object,
  // will retain current scroll position.
  let position = false

  // if no children detected
  if (to.matched.length < 2) {
    // scroll to the top of the page
    position = { x: 0, y: 0 }
  } else if (to.matched.some((r) => r.components.default.options.scrollToTop)) {
    // if one of the children has scrollToTop option set to true
    position = { x: 0, y: 0 }
  }

  // savedPosition is only available for popstate navigations (back button)
  if (savedPosition) {
    position = savedPosition
  }

  return new Promise(resolve => {
    // wait for the out transition to complete (if necessary)
    window.$nuxt.$once('triggerScroll', () => {
      // coords will be used if no selector is provided,
      // or if the selector didn't match any element.
      if (to.hash) {
        let hash = to.hash
        // CSS.escape() is not supported with IE and Edge.
        if (typeof window.CSS !== 'undefined' && typeof window.CSS.escape !== 'undefined') {
          hash = '#' + window.CSS.escape(hash.substr(1))
        }
        try {
          if (document.querySelector(hash)) {
            // scroll to anchor by returning the selector
            position = { selector: hash }
          }
        } catch (e) {
          console.warn('Failed to save scroll position. Please add CSS.escape() polyfill (https://github.com/mathiasbynens/CSS.escape).')
        }
      }
      resolve(position)
    })
  })
}


export function createRouter () {
  return new Router({
    mode: 'history',
    base: '/',
    linkActiveClass: 'nuxt-link-active',
    linkExactActiveClass: 'nuxt-link-exact-active',
    scrollBehavior,
    routes: [
		{
			path: "/Datetime",
			component: _496cae71,
			name: "Datetime"
		},
		{
			path: "/XTextarea.vue.vux",
			component: _0b14dac0,
			name: "XTextarea.vue.vux"
		},
		{
			path: "/XSwitch.vue.vux",
			component: _927be834,
			name: "XSwitch.vue.vux"
		},
		{
			path: "/PopupHeader",
			component: _6f1b7c5a,
			name: "PopupHeader"
		},
		{
			path: "/XTable.vue.vux",
			component: _bea1f280,
			name: "XTable.vue.vux"
		},
		{
			path: "/Group.vue.vux",
			component: _478ee0d3,
			name: "Group.vue.vux"
		},
		{
			path: "/Issue2365",
			component: _7969d913,
			name: "Issue2365"
		},
		{
			path: "/Issue471",
			component: _61cec6ea,
			name: "Issue471"
		},
		{
			path: "/Querystring",
			component: _0860b653,
			name: "Querystring"
		},
		{
			path: "/XTable",
			component: _6a03d32c,
			name: "XTable"
		},
		{
			path: "/Card",
			component: _ad418af4,
			name: "Card"
		},
		{
			path: "/Issue1211",
			component: _583c5058,
			name: "Issue1211"
		},
		{
			path: "/Device",
			component: _153b0b4c,
			name: "Device"
		},
		{
			path: "/Radio.vue.vux",
			component: _16e3bdd7,
			name: "Radio.vue.vux"
		},
		{
			path: "/XProgress.vue.vux",
			component: _ec6b7c66,
			name: "XProgress.vue.vux"
		},
		{
			path: "/XSwitch",
			component: _1e0facc6,
			name: "XSwitch"
		},
		{
			path: "/Divider.vue.vux",
			component: _747d8079,
			name: "Divider.vue.vux"
		},
		{
			path: "/Center",
			component: _2beda1cb,
			name: "Center"
		},
		{
			path: "/Blur",
			component: _e5732746,
			name: "Blur"
		},
		{
			path: "/Pullup.vue.vux",
			component: _d4fb11d4,
			name: "Pullup.vue.vux"
		},
		{
			path: "/XDialog.vue.vux",
			component: _21061d5c,
			name: "XDialog.vue.vux"
		},
		{
			path: "/Issue1211.vue.vux",
			component: _9cec15d0,
			name: "Issue1211.vue.vux"
		},
		{
			path: "/Toast",
			component: _b92737be,
			name: "Toast"
		},
		{
			path: "/Checker.vue.vux",
			component: _767e0a5d,
			name: "Checker.vue.vux"
		},
		{
			path: "/ButtonTab.vue.vux",
			component: _5548fa62,
			name: "ButtonTab.vue.vux"
		},
		{
			path: "/Issue2365.vue.vux",
			component: _96ce420e,
			name: "Issue2365.vue.vux"
		},
		{
			path: "/Querystring.vue.vux",
			component: _3e2acc8e,
			name: "Querystring.vue.vux"
		},
		{
			path: "/XImgPopup",
			component: _118db18a,
			name: "XImgPopup"
		},
		{
			path: "/PulldownPullup",
			component: _7855ad46,
			name: "PulldownPullup"
		},
		{
			path: "/FormatTime",
			component: _5e2dc25a,
			name: "FormatTime"
		},
		{
			path: "/Checklist.vue.vux",
			component: _0869b84c,
			name: "Checklist.vue.vux"
		},
		{
			path: "/Qrcode",
			component: _e45956f8,
			name: "Qrcode"
		},
		{
			path: "/XCircle.vue.vux",
			component: _6aa6a66c,
			name: "XCircle.vue.vux"
		},
		{
			path: "/Checklist",
			component: _5a6ef1c0,
			name: "Checklist"
		},
		{
			path: "/Flow",
			component: _22d481a4,
			name: "Flow"
		},
		{
			path: "/XImgScroller",
			component: _424ae87b,
			name: "XImgScroller"
		},
		{
			path: "/Qrcode.vue.vux",
			component: _d9f85b30,
			name: "Qrcode.vue.vux"
		},
		{
			path: "/Issue833.vue.vux",
			component: _52b41bb7,
			name: "Issue833.vue.vux"
		},
		{
			path: "/Flexbox",
			component: _f09721a8,
			name: "Flexbox"
		},
		{
			path: "/XHr",
			component: _18d2b49c,
			name: "XHr"
		},
		{
			path: "/Pulldown.vue.vux",
			component: _b3772922,
			name: "Pulldown.vue.vux"
		},
		{
			path: "/Issue649",
			component: _34374538,
			name: "Issue649"
		},
		{
			path: "/InlineCalendarEachDaySlot",
			component: _f1ab4f08,
			name: "InlineCalendarEachDaySlot"
		},
		{
			path: "/XHeader.vue.vux",
			component: _bf1b0226,
			name: "XHeader.vue.vux"
		},
		{
			path: "/Drawer",
			component: _0017bf32,
			name: "Drawer"
		},
		{
			path: "/XIcon.vue.vux",
			component: _bd8099fe,
			name: "XIcon.vue.vux"
		},
		{
			path: "/Donate.vue.vux",
			component: _402ae4e7,
			name: "Donate.vue.vux"
		},
		{
			path: "/Flexbox.vue.vux",
			component: _17522a80,
			name: "Flexbox.vue.vux"
		},
		{
			path: "/Pullup",
			component: _7c2c7796,
			name: "Pullup"
		},
		{
			path: "/PopupPicker",
			component: _04450158,
			name: "PopupPicker"
		},
		{
			path: "/XAddress.vue.vux",
			component: _17d9bc5a,
			name: "XAddress.vue.vux"
		},
		{
			path: "/ToggleText",
			component: _1514c9d2,
			name: "ToggleText"
		},
		{
			path: "/Donate",
			component: _16ae82e5,
			name: "Donate"
		},
		{
			path: "/InlineLoading.vue.vux",
			component: _b2d94b22,
			name: "InlineLoading.vue.vux"
		},
		{
			path: "/Close",
			component: _77015b72,
			name: "Close"
		},
		{
			path: "/TabbarSimple.vue.vux",
			component: _450538f4,
			name: "TabbarSimple.vue.vux"
		},
		{
			path: "/ColorPicker.vue.vux",
			component: _7fde83c1,
			name: "ColorPicker.vue.vux"
		},
		{
			path: "/PopupRadio",
			component: _c10b0fb6,
			name: "PopupRadio"
		},
		{
			path: "/OnePx",
			component: _23668cf0,
			name: "OnePx"
		},
		{
			path: "/Badge.vue.vux",
			component: _586145a2,
			name: "Badge.vue.vux"
		},
		{
			path: "/Pulldown",
			component: _f7d87546,
			name: "Pulldown"
		},
		{
			path: "/Swipeout",
			component: _c418e76c,
			name: "Swipeout"
		},
		{
			path: "/CellBox",
			component: _a4e474fa,
			name: "CellBox"
		},
		{
			path: "/InlineCalendar",
			component: _04726126,
			name: "InlineCalendar"
		},
		{
			path: "/Popover.vue.vux",
			component: _42a0c56d,
			name: "Popover.vue.vux"
		},
		{
			path: "/Confirm",
			component: _05448fba,
			name: "Confirm"
		},
		{
			path: "/Tab",
			component: _6f01770f,
			name: "Tab"
		},
		{
			path: "/XAddress",
			component: _54fa525c,
			name: "XAddress"
		},
		{
			path: "/Msg",
			component: _1bb388fb,
			name: "Msg"
		},
		{
			path: "/Demo.vue.vux",
			component: _20911173,
			name: "Demo.vue.vux"
		},
		{
			path: "/Timeline.vue.vux",
			component: _b0da9956,
			name: "Timeline.vue.vux"
		},
		{
			path: "/IconLoading.vue.vux",
			component: _2784dbcf,
			name: "IconLoading.vue.vux"
		},
		{
			path: "/Test.vue.vux",
			component: _7bd21764,
			name: "Test.vue.vux"
		},
		{
			path: "/Clocker.vue.vux",
			component: _ee4fa5d2,
			name: "Clocker.vue.vux"
		},
		{
			path: "/Spinner.vue.vux",
			component: _4d398179,
			name: "Spinner.vue.vux"
		},
		{
			path: "/DatetimeView",
			component: _823a4394,
			name: "DatetimeView"
		},
		{
			path: "/Icon.vue.vux",
			component: _191a0dc6,
			name: "Icon.vue.vux"
		},
		{
			path: "/InlineXNumber",
			component: _4ec8eefc,
			name: "InlineXNumber"
		},
		{
			path: "/InlineCalendarMulti.vue.vux",
			component: _d05e4720,
			name: "InlineCalendarMulti.vue.vux"
		},
		{
			path: "/WechatEmotion.vue.vux",
			component: _6c20813d,
			name: "WechatEmotion.vue.vux"
		},
		{
			path: "/Masker",
			component: _ddefc7e2,
			name: "Masker"
		},
		{
			path: "/Inline-calendar-start-date.vue.vux",
			component: _78df1b7c,
			name: "Inline-calendar-start-date.vue.vux"
		},
		{
			path: "/OnePx.vue.vux",
			component: _42b8fe38,
			name: "OnePx.vue.vux"
		},
		{
			path: "/Step",
			component: _01faef7c,
			name: "Step"
		},
		{
			path: "/Selector",
			component: _5ba3d656,
			name: "Selector"
		},
		{
			path: "/TabbarLink",
			component: _66ad8a6e,
			name: "TabbarLink"
		},
		{
			path: "/Previewer",
			component: _2ab9d46f,
			name: "Previewer"
		},
		{
			path: "/Cell.vue.vux",
			component: _7dc49ed8,
			name: "Cell.vue.vux"
		},
		{
			path: "/LoadMore",
			component: _68639451,
			name: "LoadMore"
		},
		{
			path: "/TabbarIcon",
			component: _18f88126,
			name: "TabbarIcon"
		},
		{
			path: "/Picker",
			component: _dc15ccf8,
			name: "Picker"
		},
		{
			path: "/InlineCalendarMulti",
			component: _72ea037c,
			name: "InlineCalendarMulti"
		},
		{
			path: "/Picker.vue.vux",
			component: _1f4a6d68,
			name: "Picker.vue.vux"
		},
		{
			path: "/DatetimeRange",
			component: _510ea8fc,
			name: "DatetimeRange"
		},
		{
			path: "/Issue702.vue.vux",
			component: _c4c8cd94,
			name: "Issue702.vue.vux"
		},
		{
			path: "/Loading",
			component: _3a2a2794,
			name: "Loading"
		},
		{
			path: "/GridNumber",
			component: _ba4280b6,
			name: "GridNumber"
		},
		{
			path: "/XImg",
			component: _30e46621,
			name: "XImg"
		},
		{
			path: "/Issue865.vue.vux",
			component: _2da079f8,
			name: "Issue865.vue.vux"
		},
		{
			path: "/Issue189",
			component: _328b7f6f,
			name: "Issue189"
		},
		{
			path: "/TabbarLink.vue.vux",
			component: _f9f1fd84,
			name: "TabbarLink.vue.vux"
		},
		{
			path: "/ScrollerHeader.vue.vux",
			component: _5db332ef,
			name: "ScrollerHeader.vue.vux"
		},
		{
			path: "/Issue865",
			component: _bd95e218,
			name: "Issue865"
		},
		{
			path: "/CheckIcon.vue.vux",
			component: _e7fd051e,
			name: "CheckIcon.vue.vux"
		},
		{
			path: "/Shake",
			component: _29859580,
			name: "Shake"
		},
		{
			path: "/Datetime.vue.vux",
			component: _854a384a,
			name: "Datetime.vue.vux"
		},
		{
			path: "/Grid.vue.vux",
			component: _d08f4fa0,
			name: "Grid.vue.vux"
		},
		{
			path: "/Scroller",
			component: _1b71efb0,
			name: "Scroller"
		},
		{
			path: "/Alert",
			component: _3c98b4d6,
			name: "Alert"
		},
		{
			path: "/Search.vue.vux",
			component: _4b3887e4,
			name: "Search.vue.vux"
		},
		{
			path: "/Range",
			component: _66f68c37,
			name: "Range"
		},
		{
			path: "/NumberRoller.vue.vux",
			component: _59e170c3,
			name: "NumberRoller.vue.vux"
		},
		{
			path: "/Range.vue.vux",
			component: _80d1c956,
			name: "Range.vue.vux"
		},
		{
			path: "/WepayInput.vue.vux",
			component: _18b196e6,
			name: "WepayInput.vue.vux"
		},
		{
			path: "/Countup.vue.vux",
			component: _6a47bf08,
			name: "Countup.vue.vux"
		},
		{
			path: "/Masker.vue.vux",
			component: _8316ef06,
			name: "Masker.vue.vux"
		},
		{
			path: "/WeekCalendar.vue.vux",
			component: _3b2656a4,
			name: "WeekCalendar.vue.vux"
		},
		{
			path: "/GridNumber.vue.vux",
			component: _4d60d027,
			name: "GridNumber.vue.vux"
		},
		{
			path: "/XButton.vue.vux",
			component: _5ddb5db0,
			name: "XButton.vue.vux"
		},
		{
			path: "/ScrollerFull.vue.vux",
			component: _1e49e0a6,
			name: "ScrollerFull.vue.vux"
		},
		{
			path: "/Countdown",
			component: _1159ce2b,
			name: "Countdown"
		},
		{
			path: "/Issue598",
			component: _07cb2d91,
			name: "Issue598"
		},
		{
			path: "/PopupHeader.vue.vux",
			component: _4fad1e39,
			name: "PopupHeader.vue.vux"
		},
		{
			path: "/DatetimeRange.vue.vux",
			component: _5baf06f0,
			name: "DatetimeRange.vue.vux"
		},
		{
			path: "/Issue598.vue.vux",
			component: _6aa520bb,
			name: "Issue598.vue.vux"
		},
		{
			path: "/Agree.vue.vux",
			component: _eb0612f4,
			name: "Agree.vue.vux"
		},
		{
			path: "/FormPreview",
			component: _29e4ff44,
			name: "FormPreview"
		},
		{
			path: "/WechatEmotion",
			component: _088e7b4f,
			name: "WechatEmotion"
		},
		{
			path: "/DatetimeView.vue.vux",
			component: _c2ad4714,
			name: "DatetimeView.vue.vux"
		},
		{
			path: "/Checker",
			component: _3d1af62f,
			name: "Checker"
		},
		{
			path: "/XProgress",
			component: _0f058682,
			name: "XProgress"
		},
		{
			path: "/Popup.vue.vux",
			component: _09fb3886,
			name: "Popup.vue.vux"
		},
		{
			path: "/Rater",
			component: _bcad1fe8,
			name: "Rater"
		},
		{
			path: "/CheckIcon",
			component: _5a72119b,
			name: "CheckIcon"
		},
		{
			path: "/Issue461.vue.vux",
			component: _f13a8400,
			name: "Issue461.vue.vux"
		},
		{
			path: "/XIcon",
			component: _289403ea,
			name: "XIcon"
		},
		{
			path: "/PopupRadio.vue.vux",
			component: _7586f0b2,
			name: "PopupRadio.vue.vux"
		},
		{
			path: "/XImgScroller.vue.vux",
			component: _7c99aade,
			name: "XImgScroller.vue.vux"
		},
		{
			path: "/Blur.vue.vux",
			component: _2517e46f,
			name: "Blur.vue.vux"
		},
		{
			path: "/Reddot",
			component: _3038730e,
			name: "Reddot"
		},
		{
			path: "/FormatTime.vue.vux",
			component: _3fc047d2,
			name: "FormatTime.vue.vux"
		},
		{
			path: "/Loading.vue.vux",
			component: _9bd4e314,
			name: "Loading.vue.vux"
		},
		{
			path: "/InlineXSwitch.vue.vux",
			component: _a9805042,
			name: "InlineXSwitch.vue.vux"
		},
		{
			path: "/XInput.vue.vux",
			component: _a0985578,
			name: "XInput.vue.vux"
		},
		{
			path: "/InlineXNumber.vue.vux",
			component: _0496c4aa,
			name: "InlineXNumber.vue.vux"
		},
		{
			path: "/Calendar.vue.vux",
			component: _1bc93c38,
			name: "Calendar.vue.vux"
		},
		{
			path: "/Comment.vue.vux",
			component: _610ffd73,
			name: "Comment.vue.vux"
		},
		{
			path: "/ColorPicker",
			component: _a166f56a,
			name: "ColorPicker"
		},
		{
			path: "/Card.vue.vux",
			component: _d9a8f3b4,
			name: "Card.vue.vux"
		},
		{
			path: "/Actionsheet",
			component: _21d4bc63,
			name: "Actionsheet"
		},
		{
			path: "/Issue702",
			component: _61e77976,
			name: "Issue702"
		},
		{
			path: "/CellFormPreview.vue.vux",
			component: _34c98360,
			name: "CellFormPreview.vue.vux"
		},
		{
			path: "/Wechat.vue.vux",
			component: _7d65dbd0,
			name: "Wechat.vue.vux"
		},
		{
			path: "/Previewer.vue.vux",
			component: _4f56b7c6,
			name: "Previewer.vue.vux"
		},
		{
			path: "/NumberRoller",
			component: _131552ee,
			name: "NumberRoller"
		},
		{
			path: "/PulldownPullup.vue.vux",
			component: _50ea876f,
			name: "PulldownPullup.vue.vux"
		},
		{
			path: "/WepayInput",
			component: _0fb681c6,
			name: "WepayInput"
		},
		{
			path: "/Issue624.vue.vux",
			component: _afb1b152,
			name: "Issue624.vue.vux"
		},
		{
			path: "/InlineXSwitch",
			component: _496f1226,
			name: "InlineXSwitch"
		},
		{
			path: "/GridComponent",
			component: _6ee9975e,
			name: "GridComponent"
		},
		{
			path: "/Issue471.vue.vux",
			component: _75e5c481,
			name: "Issue471.vue.vux"
		},
		{
			path: "/InlineCalendar.vue.vux",
			component: _91f62142,
			name: "InlineCalendar.vue.vux"
		},
		{
			path: "/Test",
			component: _65827308,
			name: "Test"
		},
		{
			path: "/Msg.vue.vux",
			component: _a0260bde,
			name: "Msg.vue.vux"
		},
		{
			path: "/Swipeout.vue.vux",
			component: _509f203c,
			name: "Swipeout.vue.vux"
		},
		{
			path: "/Popup",
			component: _6f097426,
			name: "Popup"
		},
		{
			path: "/FormPreview.vue.vux",
			component: _08d37d4e,
			name: "FormPreview.vue.vux"
		},
		{
			path: "/Marquee.vue.vux",
			component: _86cf4860,
			name: "Marquee.vue.vux"
		},
		{
			path: "/Demo",
			component: _4627c84e,
			name: "Demo"
		},
		{
			path: "/Badge",
			component: _11b14b9d,
			name: "Badge"
		},
		{
			path: "/TabbarIcon.vue.vux",
			component: _2fd9ff5f,
			name: "TabbarIcon.vue.vux"
		},
		{
			path: "/DemoList.vue.vux",
			component: _6ae24275,
			name: "DemoList.vue.vux"
		},
		{
			path: "/Timeline",
			component: _ff541792,
			name: "Timeline"
		},
		{
			path: "/IconLoading",
			component: _44f1fe06,
			name: "IconLoading"
		},
		{
			path: "/Shake.vue.vux",
			component: _57f1bdec,
			name: "Shake.vue.vux"
		},
		{
			path: "/Inline-calendar-start-date",
			component: _3d4775ea,
			name: "Inline-calendar-start-date"
		},
		{
			path: "/Scroller.vue.vux",
			component: _48848dbc,
			name: "Scroller.vue.vux"
		},
		{
			path: "/Swiper.vue.vux",
			component: _2888d184,
			name: "Swiper.vue.vux"
		},
		{
			path: "/Rater.vue.vux",
			component: _dc6da440,
			name: "Rater.vue.vux"
		},
		{
			path: "/Sticky",
			component: _bf11c542,
			name: "Sticky"
		},
		{
			path: "/Sticky.vue.vux",
			component: _c89f25a6,
			name: "Sticky.vue.vux"
		},
		{
			path: "/TabbarSimple",
			component: _28d1a926,
			name: "TabbarSimple"
		},
		{
			path: "/XHeader",
			component: _aebcf8c2,
			name: "XHeader"
		},
		{
			path: "/Step.vue.vux",
			component: _34048cea,
			name: "Step.vue.vux"
		},
		{
			path: "/Countdown.vue.vux",
			component: _5ed77ee1,
			name: "Countdown.vue.vux"
		},
		{
			path: "/SearchStatic.vue.vux",
			component: _300f6940,
			name: "SearchStatic.vue.vux"
		},
		{
			path: "/Agree",
			component: _643d13b4,
			name: "Agree"
		},
		{
			path: "/Cell",
			component: _732bcfd8,
			name: "Cell"
		},
		{
			path: "/Search",
			component: _875b4cc4,
			name: "Search"
		},
		{
			path: "/Actionsheet.vue.vux",
			component: _7bd2bcae,
			name: "Actionsheet.vue.vux"
		},
		{
			path: "/InlineCalendarEachDaySlot.vue.vux",
			component: _01123120,
			name: "InlineCalendarEachDaySlot.vue.vux"
		},
		{
			path: "/Uploader",
			component: _3a4a4038,
			name: "Uploader"
		},
		{
			path: "/Toast.vue.vux",
			component: _d3f72faa,
			name: "Toast.vue.vux"
		},
		{
			path: "/XTextarea",
			component: _5690e34c,
			name: "XTextarea"
		},
		{
			path: "/XHr.vue.vux",
			component: _add6b160,
			name: "XHr.vue.vux"
		},
		{
			path: "/ScrollerFull",
			component: _b4336a42,
			name: "ScrollerFull"
		},
		{
			path: "/XNumber",
			component: _1b62be5b,
			name: "XNumber"
		},
		{
			path: "/ScrollerSwiper.vue.vux",
			component: _16821c24,
			name: "ScrollerSwiper.vue.vux"
		},
		{
			path: "/Popover",
			component: _03c4e5c2,
			name: "Popover"
		},
		{
			path: "/Device.vue.vux",
			component: _778e6aa0,
			name: "Device.vue.vux"
		},
		{
			path: "/DemoList",
			component: _ece9add2,
			name: "DemoList"
		},
		{
			path: "/XImg.vue.vux",
			component: _33e6262b,
			name: "XImg.vue.vux"
		},
		{
			path: "/Wechat",
			component: _497b901c,
			name: "Wechat"
		},
		{
			path: "/InlineLoading",
			component: _bfb89346,
			name: "InlineLoading"
		},
		{
			path: "/Confirm.vue.vux",
			component: _4d6ca31c,
			name: "Confirm.vue.vux"
		},
		{
			path: "/Swiper",
			component: _0255b46e,
			name: "Swiper"
		},
		{
			path: "/XDialog",
			component: _3f966e4c,
			name: "XDialog"
		},
		{
			path: "/Panel",
			component: _3b66a17e,
			name: "Panel"
		},
		{
			path: "/CellFormPreview",
			component: _5e77fd9c,
			name: "CellFormPreview"
		},
		{
			path: "/Comment",
			component: _6e1eafd9,
			name: "Comment"
		},
		{
			path: "/AppTest",
			component: _4e123c66,
			name: "AppTest"
		},
		{
			path: "/Panel.vue.vux",
			component: _5548d82e,
			name: "Panel.vue.vux"
		},
		{
			path: "/ToggleText.vue.vux",
			component: _2c98df16,
			name: "ToggleText.vue.vux"
		},
		{
			path: "/WeekCalendar",
			component: _4f6a4870,
			name: "WeekCalendar"
		},
		{
			path: "/XImgPopup.vue.vux",
			component: _9276585e,
			name: "XImgPopup.vue.vux"
		},
		{
			path: "/ScrollerHeader",
			component: _ad232446,
			name: "ScrollerHeader"
		},
		{
			path: "/Group",
			component: _3b5ed879,
			name: "Group"
		},
		{
			path: "/Icon",
			component: _7de11f6f,
			name: "Icon"
		},
		{
			path: "/NotFoundComponent",
			component: _3b35be70,
			name: "NotFoundComponent"
		},
		{
			path: "/Issue649.vue.vux",
			component: _a8b52198,
			name: "Issue649.vue.vux"
		},
		{
			path: "/Selector.vue.vux",
			component: _555044f7,
			name: "Selector.vue.vux"
		},
		{
			path: "/Tabbar",
			component: _762ac154,
			name: "Tabbar"
		},
		{
			path: "/LoadMore.vue.vux",
			component: _4fcfa1fb,
			name: "LoadMore.vue.vux"
		},
		{
			path: "/Issue461",
			component: _65387828,
			name: "Issue461"
		},
		{
			path: "/Flow.vue.vux",
			component: _1beba148,
			name: "Flow.vue.vux"
		},
		{
			path: "/SearchStatic",
			component: _423540ac,
			name: "SearchStatic"
		},
		{
			path: "/AppTest.vue.vux",
			component: _a1251e02,
			name: "AppTest.vue.vux"
		},
		{
			path: "/XInput",
			component: _46489128,
			name: "XInput"
		},
		{
			path: "/XButton",
			component: _4821edc4,
			name: "XButton"
		},
		{
			path: "/Tabbar.vue.vux",
			component: _50af5b98,
			name: "Tabbar.vue.vux"
		},
		{
			path: "/Tab.vue.vux",
			component: _1d626506,
			name: "Tab.vue.vux"
		},
		{
			path: "/Uploader.vue.vux",
			component: _0feabb08,
			name: "Uploader.vue.vux"
		},
		{
			path: "/Calendar",
			component: _13e654b4,
			name: "Calendar"
		},
		{
			path: "/CellBox.vue.vux",
			component: _2af71489,
			name: "CellBox.vue.vux"
		},
		{
			path: "/Clocker",
			component: _4e25de96,
			name: "Clocker"
		},
		{
			path: "/ButtonTab",
			component: _36638c06,
			name: "ButtonTab"
		},
		{
			path: "/Spinner",
			component: _d7b9f2da,
			name: "Spinner"
		},
		{
			path: "/Issue833",
			component: _c80b53d6,
			name: "Issue833"
		},
		{
			path: "/Grid",
			component: _865b8088,
			name: "Grid"
		},
		{
			path: "/PopupPicker.vue.vux",
			component: _3c0add98,
			name: "PopupPicker.vue.vux"
		},
		{
			path: "/Radio",
			component: _59eed7f5,
			name: "Radio"
		},
		{
			path: "/XNumber.vue.vux",
			component: _1018f8b1,
			name: "XNumber.vue.vux"
		},
		{
			path: "/Issue189.vue.vux",
			component: _0746cdc6,
			name: "Issue189.vue.vux"
		},
		{
			path: "/Issue624",
			component: _30871e75,
			name: "Issue624"
		},
		{
			path: "/Marquee",
			component: _b3cddfc8,
			name: "Marquee"
		},
		{
			path: "/Thanks.vue.vux",
			component: _11065c4d,
			name: "Thanks.vue.vux"
		},
		{
			path: "/XCircle",
			component: _1ee6f73c,
			name: "XCircle"
		},
		{
			path: "/ViewBox",
			component: _170a9a40,
			name: "ViewBox"
		},
		{
			path: "/Alert.vue.vux",
			component: _6392d1d6,
			name: "Alert.vue.vux"
		},
		{
			path: "/Thanks",
			component: _afdc1b82,
			name: "Thanks"
		},
		{
			path: "/Divider",
			component: _c86030da,
			name: "Divider"
		},
		{
			path: "/Countup",
			component: _c54b4838,
			name: "Countup"
		},
		{
			path: "/ScrollerSwiper",
			component: _4f3a4648,
			name: "ScrollerSwiper"
		},
		{
			path: "/GridComponent.vue.vux",
			component: _fc57fc0a,
			name: "GridComponent.vue.vux"
		},
		{
			path: "*/Datetime/index.html",
			component: _496cae71,
			name: "Datetime_locale"
		},
		{
			path: "*/XTextarea.vue.vux/index.html",
			component: _0b14dac0,
			name: "XTextarea.vue.vux_locale"
		},
		{
			path: "*/XSwitch.vue.vux/index.html",
			component: _927be834,
			name: "XSwitch.vue.vux_locale"
		},
		{
			path: "*/PopupHeader/index.html",
			component: _6f1b7c5a,
			name: "PopupHeader_locale"
		},
		{
			path: "*/XTable.vue.vux/index.html",
			component: _bea1f280,
			name: "XTable.vue.vux_locale"
		},
		{
			path: "*/Group.vue.vux/index.html",
			component: _478ee0d3,
			name: "Group.vue.vux_locale"
		},
		{
			path: "*/Issue2365/index.html",
			component: _7969d913,
			name: "Issue2365_locale"
		},
		{
			path: "*/Issue471/index.html",
			component: _61cec6ea,
			name: "Issue471_locale"
		},
		{
			path: "*/Querystring/index.html",
			component: _0860b653,
			name: "Querystring_locale"
		},
		{
			path: "*/XTable/index.html",
			component: _6a03d32c,
			name: "XTable_locale"
		},
		{
			path: "*/Card/index.html",
			component: _ad418af4,
			name: "Card_locale"
		},
		{
			path: "*/Issue1211/index.html",
			component: _583c5058,
			name: "Issue1211_locale"
		},
		{
			path: "*/Device/index.html",
			component: _153b0b4c,
			name: "Device_locale"
		},
		{
			path: "*/Radio.vue.vux/index.html",
			component: _16e3bdd7,
			name: "Radio.vue.vux_locale"
		},
		{
			path: "*/XProgress.vue.vux/index.html",
			component: _ec6b7c66,
			name: "XProgress.vue.vux_locale"
		},
		{
			path: "*/XSwitch/index.html",
			component: _1e0facc6,
			name: "XSwitch_locale"
		},
		{
			path: "*/Divider.vue.vux/index.html",
			component: _747d8079,
			name: "Divider.vue.vux_locale"
		},
		{
			path: "*/Center/index.html",
			component: _2beda1cb,
			name: "Center_locale"
		},
		{
			path: "*/Blur/index.html",
			component: _e5732746,
			name: "Blur_locale"
		},
		{
			path: "*/Pullup.vue.vux/index.html",
			component: _d4fb11d4,
			name: "Pullup.vue.vux_locale"
		},
		{
			path: "*/XDialog.vue.vux/index.html",
			component: _21061d5c,
			name: "XDialog.vue.vux_locale"
		},
		{
			path: "*/Issue1211.vue.vux/index.html",
			component: _9cec15d0,
			name: "Issue1211.vue.vux_locale"
		},
		{
			path: "*/Toast/index.html",
			component: _b92737be,
			name: "Toast_locale"
		},
		{
			path: "*/Checker.vue.vux/index.html",
			component: _767e0a5d,
			name: "Checker.vue.vux_locale"
		},
		{
			path: "*/ButtonTab.vue.vux/index.html",
			component: _5548fa62,
			name: "ButtonTab.vue.vux_locale"
		},
		{
			path: "*/Issue2365.vue.vux/index.html",
			component: _96ce420e,
			name: "Issue2365.vue.vux_locale"
		},
		{
			path: "*/Querystring.vue.vux/index.html",
			component: _3e2acc8e,
			name: "Querystring.vue.vux_locale"
		},
		{
			path: "*/XImgPopup/index.html",
			component: _118db18a,
			name: "XImgPopup_locale"
		},
		{
			path: "*/PulldownPullup/index.html",
			component: _7855ad46,
			name: "PulldownPullup_locale"
		},
		{
			path: "*/FormatTime/index.html",
			component: _5e2dc25a,
			name: "FormatTime_locale"
		},
		{
			path: "*/Checklist.vue.vux/index.html",
			component: _0869b84c,
			name: "Checklist.vue.vux_locale"
		},
		{
			path: "*/Qrcode/index.html",
			component: _e45956f8,
			name: "Qrcode_locale"
		},
		{
			path: "*/XCircle.vue.vux/index.html",
			component: _6aa6a66c,
			name: "XCircle.vue.vux_locale"
		},
		{
			path: "*/Checklist/index.html",
			component: _5a6ef1c0,
			name: "Checklist_locale"
		},
		{
			path: "*/Flow/index.html",
			component: _22d481a4,
			name: "Flow_locale"
		},
		{
			path: "*/XImgScroller/index.html",
			component: _424ae87b,
			name: "XImgScroller_locale"
		},
		{
			path: "*/Qrcode.vue.vux/index.html",
			component: _d9f85b30,
			name: "Qrcode.vue.vux_locale"
		},
		{
			path: "*/Issue833.vue.vux/index.html",
			component: _52b41bb7,
			name: "Issue833.vue.vux_locale"
		},
		{
			path: "*/Flexbox/index.html",
			component: _f09721a8,
			name: "Flexbox_locale"
		},
		{
			path: "*/XHr/index.html",
			component: _18d2b49c,
			name: "XHr_locale"
		},
		{
			path: "*/Pulldown.vue.vux/index.html",
			component: _b3772922,
			name: "Pulldown.vue.vux_locale"
		},
		{
			path: "*/Issue649/index.html",
			component: _34374538,
			name: "Issue649_locale"
		},
		{
			path: "*/InlineCalendarEachDaySlot/index.html",
			component: _f1ab4f08,
			name: "InlineCalendarEachDaySlot_locale"
		},
		{
			path: "*/XHeader.vue.vux/index.html",
			component: _bf1b0226,
			name: "XHeader.vue.vux_locale"
		},
		{
			path: "*/Drawer/index.html",
			component: _0017bf32,
			name: "Drawer_locale"
		},
		{
			path: "*/XIcon.vue.vux/index.html",
			component: _bd8099fe,
			name: "XIcon.vue.vux_locale"
		},
		{
			path: "*/Donate.vue.vux/index.html",
			component: _402ae4e7,
			name: "Donate.vue.vux_locale"
		},
		{
			path: "*/Flexbox.vue.vux/index.html",
			component: _17522a80,
			name: "Flexbox.vue.vux_locale"
		},
		{
			path: "*/Pullup/index.html",
			component: _7c2c7796,
			name: "Pullup_locale"
		},
		{
			path: "*/PopupPicker/index.html",
			component: _04450158,
			name: "PopupPicker_locale"
		},
		{
			path: "*/XAddress.vue.vux/index.html",
			component: _17d9bc5a,
			name: "XAddress.vue.vux_locale"
		},
		{
			path: "*/ToggleText/index.html",
			component: _1514c9d2,
			name: "ToggleText_locale"
		},
		{
			path: "*/Donate/index.html",
			component: _16ae82e5,
			name: "Donate_locale"
		},
		{
			path: "*/InlineLoading.vue.vux/index.html",
			component: _b2d94b22,
			name: "InlineLoading.vue.vux_locale"
		},
		{
			path: "*/Close/index.html",
			component: _77015b72,
			name: "Close_locale"
		},
		{
			path: "*/TabbarSimple.vue.vux/index.html",
			component: _450538f4,
			name: "TabbarSimple.vue.vux_locale"
		},
		{
			path: "*/ColorPicker.vue.vux/index.html",
			component: _7fde83c1,
			name: "ColorPicker.vue.vux_locale"
		},
		{
			path: "*/PopupRadio/index.html",
			component: _c10b0fb6,
			name: "PopupRadio_locale"
		},
		{
			path: "*/OnePx/index.html",
			component: _23668cf0,
			name: "OnePx_locale"
		},
		{
			path: "*/Badge.vue.vux/index.html",
			component: _586145a2,
			name: "Badge.vue.vux_locale"
		},
		{
			path: "*/Pulldown/index.html",
			component: _f7d87546,
			name: "Pulldown_locale"
		},
		{
			path: "*/Swipeout/index.html",
			component: _c418e76c,
			name: "Swipeout_locale"
		},
		{
			path: "*/CellBox/index.html",
			component: _a4e474fa,
			name: "CellBox_locale"
		},
		{
			path: "*/InlineCalendar/index.html",
			component: _04726126,
			name: "InlineCalendar_locale"
		},
		{
			path: "*/Popover.vue.vux/index.html",
			component: _42a0c56d,
			name: "Popover.vue.vux_locale"
		},
		{
			path: "*/Confirm/index.html",
			component: _05448fba,
			name: "Confirm_locale"
		},
		{
			path: "*/Tab/index.html",
			component: _6f01770f,
			name: "Tab_locale"
		},
		{
			path: "*/XAddress/index.html",
			component: _54fa525c,
			name: "XAddress_locale"
		},
		{
			path: "*/Msg/index.html",
			component: _1bb388fb,
			name: "Msg_locale"
		},
		{
			path: "*/Demo.vue.vux/index.html",
			component: _20911173,
			name: "Demo.vue.vux_locale"
		},
		{
			path: "*/Timeline.vue.vux/index.html",
			component: _b0da9956,
			name: "Timeline.vue.vux_locale"
		},
		{
			path: "*/IconLoading.vue.vux/index.html",
			component: _2784dbcf,
			name: "IconLoading.vue.vux_locale"
		},
		{
			path: "*/Test.vue.vux/index.html",
			component: _7bd21764,
			name: "Test.vue.vux_locale"
		},
		{
			path: "*/Clocker.vue.vux/index.html",
			component: _ee4fa5d2,
			name: "Clocker.vue.vux_locale"
		},
		{
			path: "*/Spinner.vue.vux/index.html",
			component: _4d398179,
			name: "Spinner.vue.vux_locale"
		},
		{
			path: "*/DatetimeView/index.html",
			component: _823a4394,
			name: "DatetimeView_locale"
		},
		{
			path: "*/Icon.vue.vux/index.html",
			component: _191a0dc6,
			name: "Icon.vue.vux_locale"
		},
		{
			path: "*/InlineXNumber/index.html",
			component: _4ec8eefc,
			name: "InlineXNumber_locale"
		},
		{
			path: "*/InlineCalendarMulti.vue.vux/index.html",
			component: _d05e4720,
			name: "InlineCalendarMulti.vue.vux_locale"
		},
		{
			path: "*/WechatEmotion.vue.vux/index.html",
			component: _6c20813d,
			name: "WechatEmotion.vue.vux_locale"
		},
		{
			path: "*/Masker/index.html",
			component: _ddefc7e2,
			name: "Masker_locale"
		},
		{
			path: "*/Inline-calendar-start-date.vue.vux/index.html",
			component: _78df1b7c,
			name: "Inline-calendar-start-date.vue.vux_locale"
		},
		{
			path: "*/OnePx.vue.vux/index.html",
			component: _42b8fe38,
			name: "OnePx.vue.vux_locale"
		},
		{
			path: "*/Step/index.html",
			component: _01faef7c,
			name: "Step_locale"
		},
		{
			path: "*/Selector/index.html",
			component: _5ba3d656,
			name: "Selector_locale"
		},
		{
			path: "*/TabbarLink/index.html",
			component: _66ad8a6e,
			name: "TabbarLink_locale"
		},
		{
			path: "*/Previewer/index.html",
			component: _2ab9d46f,
			name: "Previewer_locale"
		},
		{
			path: "*/Cell.vue.vux/index.html",
			component: _7dc49ed8,
			name: "Cell.vue.vux_locale"
		},
		{
			path: "*/LoadMore/index.html",
			component: _68639451,
			name: "LoadMore_locale"
		},
		{
			path: "*/TabbarIcon/index.html",
			component: _18f88126,
			name: "TabbarIcon_locale"
		},
		{
			path: "*/Picker/index.html",
			component: _dc15ccf8,
			name: "Picker_locale"
		},
		{
			path: "*/InlineCalendarMulti/index.html",
			component: _72ea037c,
			name: "InlineCalendarMulti_locale"
		},
		{
			path: "*/Picker.vue.vux/index.html",
			component: _1f4a6d68,
			name: "Picker.vue.vux_locale"
		},
		{
			path: "*/DatetimeRange/index.html",
			component: _510ea8fc,
			name: "DatetimeRange_locale"
		},
		{
			path: "*/Issue702.vue.vux/index.html",
			component: _c4c8cd94,
			name: "Issue702.vue.vux_locale"
		},
		{
			path: "*/Loading/index.html",
			component: _3a2a2794,
			name: "Loading_locale"
		},
		{
			path: "*/GridNumber/index.html",
			component: _ba4280b6,
			name: "GridNumber_locale"
		},
		{
			path: "*/XImg/index.html",
			component: _30e46621,
			name: "XImg_locale"
		},
		{
			path: "*/Issue865.vue.vux/index.html",
			component: _2da079f8,
			name: "Issue865.vue.vux_locale"
		},
		{
			path: "*/Issue189/index.html",
			component: _328b7f6f,
			name: "Issue189_locale"
		},
		{
			path: "*/TabbarLink.vue.vux/index.html",
			component: _f9f1fd84,
			name: "TabbarLink.vue.vux_locale"
		},
		{
			path: "*/ScrollerHeader.vue.vux/index.html",
			component: _5db332ef,
			name: "ScrollerHeader.vue.vux_locale"
		},
		{
			path: "*/Issue865/index.html",
			component: _bd95e218,
			name: "Issue865_locale"
		},
		{
			path: "*/CheckIcon.vue.vux/index.html",
			component: _e7fd051e,
			name: "CheckIcon.vue.vux_locale"
		},
		{
			path: "*/Shake/index.html",
			component: _29859580,
			name: "Shake_locale"
		},
		{
			path: "*/Datetime.vue.vux/index.html",
			component: _854a384a,
			name: "Datetime.vue.vux_locale"
		},
		{
			path: "*/Grid.vue.vux/index.html",
			component: _d08f4fa0,
			name: "Grid.vue.vux_locale"
		},
		{
			path: "*/Scroller/index.html",
			component: _1b71efb0,
			name: "Scroller_locale"
		},
		{
			path: "*/Alert/index.html",
			component: _3c98b4d6,
			name: "Alert_locale"
		},
		{
			path: "*/Search.vue.vux/index.html",
			component: _4b3887e4,
			name: "Search.vue.vux_locale"
		},
		{
			path: "*/Range/index.html",
			component: _66f68c37,
			name: "Range_locale"
		},
		{
			path: "*/NumberRoller.vue.vux/index.html",
			component: _59e170c3,
			name: "NumberRoller.vue.vux_locale"
		},
		{
			path: "*/Range.vue.vux/index.html",
			component: _80d1c956,
			name: "Range.vue.vux_locale"
		},
		{
			path: "*/WepayInput.vue.vux/index.html",
			component: _18b196e6,
			name: "WepayInput.vue.vux_locale"
		},
		{
			path: "*/Countup.vue.vux/index.html",
			component: _6a47bf08,
			name: "Countup.vue.vux_locale"
		},
		{
			path: "*/Masker.vue.vux/index.html",
			component: _8316ef06,
			name: "Masker.vue.vux_locale"
		},
		{
			path: "*/WeekCalendar.vue.vux/index.html",
			component: _3b2656a4,
			name: "WeekCalendar.vue.vux_locale"
		},
		{
			path: "*/GridNumber.vue.vux/index.html",
			component: _4d60d027,
			name: "GridNumber.vue.vux_locale"
		},
		{
			path: "*/XButton.vue.vux/index.html",
			component: _5ddb5db0,
			name: "XButton.vue.vux_locale"
		},
		{
			path: "*/ScrollerFull.vue.vux/index.html",
			component: _1e49e0a6,
			name: "ScrollerFull.vue.vux_locale"
		},
		{
			path: "*/Countdown/index.html",
			component: _1159ce2b,
			name: "Countdown_locale"
		},
		{
			path: "*/Issue598/index.html",
			component: _07cb2d91,
			name: "Issue598_locale"
		},
		{
			path: "*/PopupHeader.vue.vux/index.html",
			component: _4fad1e39,
			name: "PopupHeader.vue.vux_locale"
		},
		{
			path: "*/DatetimeRange.vue.vux/index.html",
			component: _5baf06f0,
			name: "DatetimeRange.vue.vux_locale"
		},
		{
			path: "*/Issue598.vue.vux/index.html",
			component: _6aa520bb,
			name: "Issue598.vue.vux_locale"
		},
		{
			path: "*/Agree.vue.vux/index.html",
			component: _eb0612f4,
			name: "Agree.vue.vux_locale"
		},
		{
			path: "*/FormPreview/index.html",
			component: _29e4ff44,
			name: "FormPreview_locale"
		},
		{
			path: "*/WechatEmotion/index.html",
			component: _088e7b4f,
			name: "WechatEmotion_locale"
		},
		{
			path: "*/DatetimeView.vue.vux/index.html",
			component: _c2ad4714,
			name: "DatetimeView.vue.vux_locale"
		},
		{
			path: "*/Checker/index.html",
			component: _3d1af62f,
			name: "Checker_locale"
		},
		{
			path: "*/XProgress/index.html",
			component: _0f058682,
			name: "XProgress_locale"
		},
		{
			path: "*/Popup.vue.vux/index.html",
			component: _09fb3886,
			name: "Popup.vue.vux_locale"
		},
		{
			path: "*/Rater/index.html",
			component: _bcad1fe8,
			name: "Rater_locale"
		},
		{
			path: "*/CheckIcon/index.html",
			component: _5a72119b,
			name: "CheckIcon_locale"
		},
		{
			path: "*/Issue461.vue.vux/index.html",
			component: _f13a8400,
			name: "Issue461.vue.vux_locale"
		},
		{
			path: "*/XIcon/index.html",
			component: _289403ea,
			name: "XIcon_locale"
		},
		{
			path: "*/PopupRadio.vue.vux/index.html",
			component: _7586f0b2,
			name: "PopupRadio.vue.vux_locale"
		},
		{
			path: "*/XImgScroller.vue.vux/index.html",
			component: _7c99aade,
			name: "XImgScroller.vue.vux_locale"
		},
		{
			path: "*/Blur.vue.vux/index.html",
			component: _2517e46f,
			name: "Blur.vue.vux_locale"
		},
		{
			path: "*/Reddot/index.html",
			component: _3038730e,
			name: "Reddot_locale"
		},
		{
			path: "*/FormatTime.vue.vux/index.html",
			component: _3fc047d2,
			name: "FormatTime.vue.vux_locale"
		},
		{
			path: "*/Loading.vue.vux/index.html",
			component: _9bd4e314,
			name: "Loading.vue.vux_locale"
		},
		{
			path: "*/InlineXSwitch.vue.vux/index.html",
			component: _a9805042,
			name: "InlineXSwitch.vue.vux_locale"
		},
		{
			path: "*/XInput.vue.vux/index.html",
			component: _a0985578,
			name: "XInput.vue.vux_locale"
		},
		{
			path: "*/InlineXNumber.vue.vux/index.html",
			component: _0496c4aa,
			name: "InlineXNumber.vue.vux_locale"
		},
		{
			path: "*/Calendar.vue.vux/index.html",
			component: _1bc93c38,
			name: "Calendar.vue.vux_locale"
		},
		{
			path: "*/Comment.vue.vux/index.html",
			component: _610ffd73,
			name: "Comment.vue.vux_locale"
		},
		{
			path: "*/ColorPicker/index.html",
			component: _a166f56a,
			name: "ColorPicker_locale"
		},
		{
			path: "*/Card.vue.vux/index.html",
			component: _d9a8f3b4,
			name: "Card.vue.vux_locale"
		},
		{
			path: "*/Actionsheet/index.html",
			component: _21d4bc63,
			name: "Actionsheet_locale"
		},
		{
			path: "*/Issue702/index.html",
			component: _61e77976,
			name: "Issue702_locale"
		},
		{
			path: "*/CellFormPreview.vue.vux/index.html",
			component: _34c98360,
			name: "CellFormPreview.vue.vux_locale"
		},
		{
			path: "*/Wechat.vue.vux/index.html",
			component: _7d65dbd0,
			name: "Wechat.vue.vux_locale"
		},
		{
			path: "*/Previewer.vue.vux/index.html",
			component: _4f56b7c6,
			name: "Previewer.vue.vux_locale"
		},
		{
			path: "*/NumberRoller/index.html",
			component: _131552ee,
			name: "NumberRoller_locale"
		},
		{
			path: "*/PulldownPullup.vue.vux/index.html",
			component: _50ea876f,
			name: "PulldownPullup.vue.vux_locale"
		},
		{
			path: "*/WepayInput/index.html",
			component: _0fb681c6,
			name: "WepayInput_locale"
		},
		{
			path: "*/Issue624.vue.vux/index.html",
			component: _afb1b152,
			name: "Issue624.vue.vux_locale"
		},
		{
			path: "*/InlineXSwitch/index.html",
			component: _496f1226,
			name: "InlineXSwitch_locale"
		},
		{
			path: "*/GridComponent/index.html",
			component: _6ee9975e,
			name: "GridComponent_locale"
		},
		{
			path: "*/Issue471.vue.vux/index.html",
			component: _75e5c481,
			name: "Issue471.vue.vux_locale"
		},
		{
			path: "*/InlineCalendar.vue.vux/index.html",
			component: _91f62142,
			name: "InlineCalendar.vue.vux_locale"
		},
		{
			path: "*/Test/index.html",
			component: _65827308,
			name: "Test_locale"
		},
		{
			path: "*/Msg.vue.vux/index.html",
			component: _a0260bde,
			name: "Msg.vue.vux_locale"
		},
		{
			path: "*/Swipeout.vue.vux/index.html",
			component: _509f203c,
			name: "Swipeout.vue.vux_locale"
		},
		{
			path: "*/Popup/index.html",
			component: _6f097426,
			name: "Popup_locale"
		},
		{
			path: "*/FormPreview.vue.vux/index.html",
			component: _08d37d4e,
			name: "FormPreview.vue.vux_locale"
		},
		{
			path: "*/Marquee.vue.vux/index.html",
			component: _86cf4860,
			name: "Marquee.vue.vux_locale"
		},
		{
			path: "*/Demo/index.html",
			component: _4627c84e,
			name: "Demo_locale"
		},
		{
			path: "*/Badge/index.html",
			component: _11b14b9d,
			name: "Badge_locale"
		},
		{
			path: "*/TabbarIcon.vue.vux/index.html",
			component: _2fd9ff5f,
			name: "TabbarIcon.vue.vux_locale"
		},
		{
			path: "*/DemoList.vue.vux/index.html",
			component: _6ae24275,
			name: "DemoList.vue.vux_locale"
		},
		{
			path: "*/Timeline/index.html",
			component: _ff541792,
			name: "Timeline_locale"
		},
		{
			path: "*/IconLoading/index.html",
			component: _44f1fe06,
			name: "IconLoading_locale"
		},
		{
			path: "*/Shake.vue.vux/index.html",
			component: _57f1bdec,
			name: "Shake.vue.vux_locale"
		},
		{
			path: "*/Inline-calendar-start-date/index.html",
			component: _3d4775ea,
			name: "Inline-calendar-start-date_locale"
		},
		{
			path: "*/Scroller.vue.vux/index.html",
			component: _48848dbc,
			name: "Scroller.vue.vux_locale"
		},
		{
			path: "*/Swiper.vue.vux/index.html",
			component: _2888d184,
			name: "Swiper.vue.vux_locale"
		},
		{
			path: "*/Rater.vue.vux/index.html",
			component: _dc6da440,
			name: "Rater.vue.vux_locale"
		},
		{
			path: "*/Sticky/index.html",
			component: _bf11c542,
			name: "Sticky_locale"
		},
		{
			path: "*/Sticky.vue.vux/index.html",
			component: _c89f25a6,
			name: "Sticky.vue.vux_locale"
		},
		{
			path: "*/TabbarSimple/index.html",
			component: _28d1a926,
			name: "TabbarSimple_locale"
		},
		{
			path: "*/XHeader/index.html",
			component: _aebcf8c2,
			name: "XHeader_locale"
		},
		{
			path: "*/Step.vue.vux/index.html",
			component: _34048cea,
			name: "Step.vue.vux_locale"
		},
		{
			path: "*/Countdown.vue.vux/index.html",
			component: _5ed77ee1,
			name: "Countdown.vue.vux_locale"
		},
		{
			path: "*/SearchStatic.vue.vux/index.html",
			component: _300f6940,
			name: "SearchStatic.vue.vux_locale"
		},
		{
			path: "*/Agree/index.html",
			component: _643d13b4,
			name: "Agree_locale"
		},
		{
			path: "*/Cell/index.html",
			component: _732bcfd8,
			name: "Cell_locale"
		},
		{
			path: "*/Search/index.html",
			component: _875b4cc4,
			name: "Search_locale"
		},
		{
			path: "*/Actionsheet.vue.vux/index.html",
			component: _7bd2bcae,
			name: "Actionsheet.vue.vux_locale"
		},
		{
			path: "*/InlineCalendarEachDaySlot.vue.vux/index.html",
			component: _01123120,
			name: "InlineCalendarEachDaySlot.vue.vux_locale"
		},
		{
			path: "*/Uploader/index.html",
			component: _3a4a4038,
			name: "Uploader_locale"
		},
		{
			path: "*/Toast.vue.vux/index.html",
			component: _d3f72faa,
			name: "Toast.vue.vux_locale"
		},
		{
			path: "*/XTextarea/index.html",
			component: _5690e34c,
			name: "XTextarea_locale"
		},
		{
			path: "*/XHr.vue.vux/index.html",
			component: _add6b160,
			name: "XHr.vue.vux_locale"
		},
		{
			path: "*/ScrollerFull/index.html",
			component: _b4336a42,
			name: "ScrollerFull_locale"
		},
		{
			path: "*/XNumber/index.html",
			component: _1b62be5b,
			name: "XNumber_locale"
		},
		{
			path: "*/ScrollerSwiper.vue.vux/index.html",
			component: _16821c24,
			name: "ScrollerSwiper.vue.vux_locale"
		},
		{
			path: "*/Popover/index.html",
			component: _03c4e5c2,
			name: "Popover_locale"
		},
		{
			path: "*/Device.vue.vux/index.html",
			component: _778e6aa0,
			name: "Device.vue.vux_locale"
		},
		{
			path: "*/DemoList/index.html",
			component: _ece9add2,
			name: "DemoList_locale"
		},
		{
			path: "*/XImg.vue.vux/index.html",
			component: _33e6262b,
			name: "XImg.vue.vux_locale"
		},
		{
			path: "*/Wechat/index.html",
			component: _497b901c,
			name: "Wechat_locale"
		},
		{
			path: "*/InlineLoading/index.html",
			component: _bfb89346,
			name: "InlineLoading_locale"
		},
		{
			path: "*/Confirm.vue.vux/index.html",
			component: _4d6ca31c,
			name: "Confirm.vue.vux_locale"
		},
		{
			path: "*/Swiper/index.html",
			component: _0255b46e,
			name: "Swiper_locale"
		},
		{
			path: "*/XDialog/index.html",
			component: _3f966e4c,
			name: "XDialog_locale"
		},
		{
			path: "*/Panel/index.html",
			component: _3b66a17e,
			name: "Panel_locale"
		},
		{
			path: "*/CellFormPreview/index.html",
			component: _5e77fd9c,
			name: "CellFormPreview_locale"
		},
		{
			path: "*/Comment/index.html",
			component: _6e1eafd9,
			name: "Comment_locale"
		},
		{
			path: "*/AppTest/index.html",
			component: _4e123c66,
			name: "AppTest_locale"
		},
		{
			path: "*/Panel.vue.vux/index.html",
			component: _5548d82e,
			name: "Panel.vue.vux_locale"
		},
		{
			path: "*/ToggleText.vue.vux/index.html",
			component: _2c98df16,
			name: "ToggleText.vue.vux_locale"
		},
		{
			path: "*/WeekCalendar/index.html",
			component: _4f6a4870,
			name: "WeekCalendar_locale"
		},
		{
			path: "*/XImgPopup.vue.vux/index.html",
			component: _9276585e,
			name: "XImgPopup.vue.vux_locale"
		},
		{
			path: "*/ScrollerHeader/index.html",
			component: _ad232446,
			name: "ScrollerHeader_locale"
		},
		{
			path: "*/Group/index.html",
			component: _3b5ed879,
			name: "Group_locale"
		},
		{
			path: "*/Icon/index.html",
			component: _7de11f6f,
			name: "Icon_locale"
		},
		{
			path: "*/NotFoundComponent/index.html",
			component: _3b35be70,
			name: "NotFoundComponent_locale"
		},
		{
			path: "*/Issue649.vue.vux/index.html",
			component: _a8b52198,
			name: "Issue649.vue.vux_locale"
		},
		{
			path: "*/Selector.vue.vux/index.html",
			component: _555044f7,
			name: "Selector.vue.vux_locale"
		},
		{
			path: "*/Tabbar/index.html",
			component: _762ac154,
			name: "Tabbar_locale"
		},
		{
			path: "*/LoadMore.vue.vux/index.html",
			component: _4fcfa1fb,
			name: "LoadMore.vue.vux_locale"
		},
		{
			path: "*/Issue461/index.html",
			component: _65387828,
			name: "Issue461_locale"
		},
		{
			path: "*/Flow.vue.vux/index.html",
			component: _1beba148,
			name: "Flow.vue.vux_locale"
		},
		{
			path: "*/SearchStatic/index.html",
			component: _423540ac,
			name: "SearchStatic_locale"
		},
		{
			path: "*/AppTest.vue.vux/index.html",
			component: _a1251e02,
			name: "AppTest.vue.vux_locale"
		},
		{
			path: "*/XInput/index.html",
			component: _46489128,
			name: "XInput_locale"
		},
		{
			path: "*/XButton/index.html",
			component: _4821edc4,
			name: "XButton_locale"
		},
		{
			path: "*/Tabbar.vue.vux/index.html",
			component: _50af5b98,
			name: "Tabbar.vue.vux_locale"
		},
		{
			path: "*/Tab.vue.vux/index.html",
			component: _1d626506,
			name: "Tab.vue.vux_locale"
		},
		{
			path: "*/Uploader.vue.vux/index.html",
			component: _0feabb08,
			name: "Uploader.vue.vux_locale"
		},
		{
			path: "*/Calendar/index.html",
			component: _13e654b4,
			name: "Calendar_locale"
		},
		{
			path: "*/CellBox.vue.vux/index.html",
			component: _2af71489,
			name: "CellBox.vue.vux_locale"
		},
		{
			path: "*/Clocker/index.html",
			component: _4e25de96,
			name: "Clocker_locale"
		},
		{
			path: "*/ButtonTab/index.html",
			component: _36638c06,
			name: "ButtonTab_locale"
		},
		{
			path: "*/Spinner/index.html",
			component: _d7b9f2da,
			name: "Spinner_locale"
		},
		{
			path: "*/Issue833/index.html",
			component: _c80b53d6,
			name: "Issue833_locale"
		},
		{
			path: "*/Grid/index.html",
			component: _865b8088,
			name: "Grid_locale"
		},
		{
			path: "*/PopupPicker.vue.vux/index.html",
			component: _3c0add98,
			name: "PopupPicker.vue.vux_locale"
		},
		{
			path: "*/Radio/index.html",
			component: _59eed7f5,
			name: "Radio_locale"
		},
		{
			path: "*/XNumber.vue.vux/index.html",
			component: _1018f8b1,
			name: "XNumber.vue.vux_locale"
		},
		{
			path: "*/Issue189.vue.vux/index.html",
			component: _0746cdc6,
			name: "Issue189.vue.vux_locale"
		},
		{
			path: "*/Issue624/index.html",
			component: _30871e75,
			name: "Issue624_locale"
		},
		{
			path: "*/Marquee/index.html",
			component: _b3cddfc8,
			name: "Marquee_locale"
		},
		{
			path: "*/Thanks.vue.vux/index.html",
			component: _11065c4d,
			name: "Thanks.vue.vux_locale"
		},
		{
			path: "*/XCircle/index.html",
			component: _1ee6f73c,
			name: "XCircle_locale"
		},
		{
			path: "*/ViewBox/index.html",
			component: _170a9a40,
			name: "ViewBox_locale"
		},
		{
			path: "*/Alert.vue.vux/index.html",
			component: _6392d1d6,
			name: "Alert.vue.vux_locale"
		},
		{
			path: "*/Thanks/index.html",
			component: _afdc1b82,
			name: "Thanks_locale"
		},
		{
			path: "*/Divider/index.html",
			component: _c86030da,
			name: "Divider_locale"
		},
		{
			path: "*/Countup/index.html",
			component: _c54b4838,
			name: "Countup_locale"
		},
		{
			path: "*/ScrollerSwiper/index.html",
			component: _4f3a4648,
			name: "ScrollerSwiper_locale"
		},
		{
			path: "*/GridComponent.vue.vux/index.html",
			component: _fc57fc0a,
			name: "GridComponent.vue.vux_locale"
		}
    ],
    
    
    fallback: false
  })
}
